ckan/config/environment.py

```diff
@@ -292,6 +292,7 @@ def update_config():
 
     # CONFIGURATION OPTIONS HERE (note: all config options will override
     # any Pylons config options)
+    config.setdefault('pool_pre_ping', True)
```

Comment:
A one-line comment explaining this wouldn't go amiss.

Reply:
Sure, it makes sense.